File: Selenium/Activities/Activity19.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait

# Start the Driver
with webdriver.Firefox() as driver:
    # Declare the wait variable
    wait = WebDriverWait(driver, timeout=20)

    # Navigate to the URL
    driver.get("https://training-support.net/webelements/alerts")

    # Print the title of the page
    print("Page title is: ", driver.title)

    # Find and click the button to open the alert
    driver.find_element(By.ID, "confirmation").click()

    # Switch focus to the alert
    confirmAlert = wait.until(EC.alert_is_present())

    # Print the text in the alert
    alertText = confirmAlert.text
    print("Text in alert: " + alertText)

    # Close the alert by clicking OK
    confirmAlert.dismiss()

    # Print the message
    print(driver.find_element(By.ID, "result").text)

    driver.quit()

File: Selenium/Activities/Activity19.java
package activities;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;


public class Activity19 {
    public static void main(String[] args) {
    	 WebDriver driver = new ChromeDriver();
    	    
         driver.get("https://training-support.net/webelements/alerts");
         System.out.println("Page title: " + driver.getTitle());

        
        driver.findElement(By.id("confirm")).click();

      
        Alert confirmAlert = driver.switchTo().alert();

        
        String alertText = confirmAlert.getText();
        System.out.println("Text in alert: " + alertText);

        confirmAlert.accept();

        // Can also close the alert by clicking Cancel
        // confirmAlert.dismiss();

        System.out.println(driver.findElement(By.id("result")).getText());

      
        driver.quit();
    }
}